.github/workflows/main.yml

```diff
@@ -161,7 +161,7 @@ jobs:
         run: $SBT test
 
       - name: Run Integration Tests
-        run: $SBT IntegrationTest/test
+        run: $SBT daffodil-test-integration/Test/test
```

Comment:
According to the DEVELOP.md changes below, `daffodil-test-integration/test` would work too.  Why do you use the longer `daffodil-test-integration/Test/test` in the CI workflow?

Reply:
Good call, the `Test` isn't needed, will remove.